### PR TITLE
Update Builder.php

### PR DESCRIPTION
**Pull request by uzrnem**
GroupBy will return Count

### CODE REVIEW

**Comment by uzrnem** [comment]
@DenchikBY Can you please look into this

File: src/MongoDB/Query/Builder.php [hunk 1]
```diff
@@ -145,6 +145,11 @@ public function groupBy($group)
         return $this;
     }
 
+    public function groupByWithCount($group)
```
Comment (DenchikBY) [inline src/MongoDB/Query/Builder.php]: Add please PHPDoc comment

Comment (DenchikBY) [reply src/MongoDB/Query/Builder.php]: Method should return $this. And it should be added to Model.php PHPDoc according last changes

**Comment by uzrnem** [comment]
Please review mow and tell me what should be added inside Model.php

**Comment by DenchikBY** [comment]
@uzrnem look into my master, now in Model.php present comment like
```
/**
  * @property UTCDateTime created_at
  * @property UTCDateTime updated_at
  *
  * @method static Builder columns(array $fields)
  * @method static Builder join(string $relationName)
  * @method static Builder where(string $field, string $operation = null, $value = null)
  * @method static Builder orWhere(string $field, string $operation = null, $value = null)
  * @method static Builder betweenWhere(string $field, int $minimum, int $maximum)
  * @method static Builder notBetweenWhere(string $field, int $minimum, int $maximum)
  * ...
```
you should update your fork and add same annotation for IDE autocomplete

**Comment by uzrnem** [comment]
Can You Please check now?

**Pull request merged**